domainlab/mk_exp.py

```diff
@@ -10,10 +10,15 @@
 
 
 
-def mk_exp(task, model, trainer, test_domain, batchsize):
+def mk_exp(task, model, trainer:str, test_domain:str, batchsize:int):
     """
-    test_domain must be a single string
+    Creates a custom experiment. The user can specify the input parameters.
+
+    Parameters: task, model, trainer (string), test_domain (string), batch size (int)
```

Comment:
here we also need a concrete example for the user to understand what kind of arguments task and model , maybe link it to an independent unit test file?